src/applications/vaos/components/TextareaWidget.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+function TextareaWidget({
```

Comment:
This is copied from the DSVA fork of RJSF, because it's one widget we haven't customized at all. I think this would be good to put in the platform code eventually, but for now, the only change I've made is to respect maxLength and show a counter